PTL converts 'Resv_ID' to 'id' when querying a reservation

Klovely opened the pull request:
#### Describe Bug or Feature
PTL converts 'Resv_ID' to 'id' when querying a reservation
When doing a status() on a reservation, PTL is supposed to return a dict with all the reservation attributes and their values, but it renames the attribute 'Resv ID' to 'id':

#### Describe Your Change
Added condition to handle above isssue in pbs_testlib.py

#### Attach Test and Valgrind Logs/Output
[smokeresult.txt](https://github.com/PBSPro/pbspro/files/3723103/smokeresult.txt)
[test_attrib_ResvID_after_fix_log_from_terminal.txt](https://github.com/PBSPro/pbspro/files/3723104/test_attrib_ResvID_after_fix_log_from_terminal.txt)
[test_attrib_ResvID_before_fix_log_from_terminal.txt](https://github.com/PBSPro/pbspro/files/3723105/test_attrib_ResvID_before_fix_log_from_terminal.txt)

hirenvadalia commented on test/tests/selftest/pbs_resvid_test.py:
```diff
@@ -0,0 +1,78 @@
+        rid = self.server.submit(r)
+        val = self.server.status(RESV, id=rid)
+        self.logger.info("Got value as list")
+        print(val)
```
use self.logger.info() instead print()

Klovely commented:
Latest log : -
=======
latest_log2_test_attrib_ResvID.txt

Klovely commented:
@prakashcv13 , I addressed your comment in latest commit please have a look .

Latest log : -
[latest_log3_test_attrib_ResvID.txt](https://github.com/PBSPro/pbspro/files/3723595/latest_log3_test_attrib_ResvID.txt)
